### ISSUE TITLE
Invalid backend response while deleting very large directory

### ISSUE THREAD

**Issue by jaimbh** [Studio-42/elFinder#3542]
Deleting a 3gb and a 4gb directory gives below error:
Invalid backend response.
Error occurred on the server side.
HTTP error 500

The directory has sub-directories and thousands of files. Deleting them individually by entering the main directory works. But deleting the main directory gives the error after some time. Moving some large sub-directories into some other directory gives same error.

The localRmdirRecursive function in elFinderVolumeDriver.class.php is not getting called at all. I checked that by inserting "file_put_contents('/home/user/abc', '12345');" in that function. The function gets called only for smaller directories. The "exec('rm -rf ' . escapeshellarg($dir), $o, $r);" line gets executed for them.

I checked in almalinux 8 and centos 7.

**Comment by dleffler** [comment]
Just a guess, but the php command took longer (timed out) than the server setting allowed which crashed php and sent the error back to the ajax call from elFinder

**Comment by jaimbh** [comment]
In which file is this ajax call getting called?

**Comment by jaimbh** [comment]
Solved the delete problem by adding 'tmbPath' => '' in the array $opts['roots'] in elfinder.php.

But when I try to move very large directory into another very large directory, it is still giving me the same error which is:
Invalid backend response.
Error occurred on the server side.
HTTP error 500

The move function in elfinderVolumeDriver.class.php is not getting called for moving of large directories. I checked by inserting file_put_contents('/home/user', '123'); inside the function. It gets called for smaller directories.
